src/base/bittorrent/torrenthandle.cpp

```diff
@@ -1950,7 +2042,14 @@ void TorrentHandle::setDownloadLimit(const int limit)
 
 void TorrentHandle::setSuperSeeding(const bool enable)
 {
-    m_nativeHandle.super_seeding(enable);
+#if (LIBTORRENT_VERSION_NUM < 10200)
+    m_nativeStatus.super_seeding = enable;
```

Comment:
Looks like you got m_nativeStatus and m_nativeHandle mixed up in that PR (for the several times)... Please recheck it carefully.

Reply:
Not quite following what you mean. This change here is to match `TorrentHandle::superSeeding()`.
And it is the preferred way because it is marked deprecated in torrent_handle: https://github.com/arvidn/libtorrent/blob/RC_1_1/include/libtorrent/torrent_handle.hpp#L970
And not in torrent_status: https://github.com/arvidn/libtorrent/blob/RC_1_1/include/libtorrent/torrent_status.hpp#L457

As for libt 1.2 APIs, there is only one way to invoke them without triggering deprecated messages so they should be correct.

Reply:
>Not quite following what you mean.

`m_nativeHandle.super_seeding(enable);` does the job but `m_nativeStatus.super_seeding = enable;` does nothing! And so on...

Reply:
Thanks, reverted.